packages/scene-composer/src/components/three-fiber/EditorCamera.tsx

```diff
@@ -243,7 +269,12 @@ export function findBestViewingPosition(
     Number.isFinite(objectBoundingBox.max.z)
   ) {
     // normal case
-    const camera = controls.object as THREE.PerspectiveCamera;
+    let camera;
```

Comment:
Nit-pick: This would be a great opportunity to use a ternary... unless you need it to have a third case of undefined. 

```
const camera = controls instanceof OrbitControlsImpl || controls instanceof MapControlsImpl
  ? controls.object as THREE. PerspectiveCamera
  : controls.camera

Reply:
Undefined is already guarded by the if above so I'll swap to a ternary.